sklearn/model_selection/tests/test_split.py

```diff
@@ -1026,6 +1026,42 @@ def test_train_test_split_errors():
     assert_raises(ValueError, train_test_split, range(3), range(42))
     assert_raises(ValueError, train_test_split, range(10),
                   shuffle=False, stratify=True)
+    assert_raises(ValueError, train_test_split, range(10),
```

Comment:
Please parameterize this test with `@pytest.mark.parametrize`

Reply:
I switched to a parameterized test. As far as I understand, I couldn't use two stacked parameterizations (one for `train_size` and one for `test_size`) as not all pair should fail. Therefore, I simply listed all the pairs I want to test.